firmware/lm32/heartbeat.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <generated/csr.h>
+#include <generated/mem.h>
+#include <hw/flags.h>
+#include <system.h>
+#include <time.h>
+
+#define HDMI_IN0_FRAMEBUFFERS_BASE 	0x01000000
```

Comment:
Don't redefine these here, use the defines from the other files (move them to the header if needed).